### PR TITLE
Added data and formats for Italian culture

### PR DESCRIPTION
**Pull request by supix**

### CODE REVIEW

File: src/Faker/Resources/Company.it-IT.resx [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+  </data>
+  <data name="Suffix" xml:space="preserve">
+    <value>s.r.l.;s.r.l.;s.r.l.;s.p.a.;s.p.a.;s.p.a.;s.c.a.r.l.;onlus</value>
+  </data>
+</root>
```
Comment (AdmiringWorm) [inline src/Faker/Resources/Company.it-IT.resx:123]: With the exception of the Suffix value, this file seem like a duplication of the english language.
If it is, could you please just remove them as there is no need to add them into a new resource file (ResourceManager should fallback to the default language (English) if no matching resource name was found

**Comment by supix** [comment]
Applying small tuning to company formats... Will commit in a while.

**Review by AdmiringWorm — APPROVED** [review]
LGTM 👍 
Thanks for your contribution, I'll merge it when all checks passes

**Comment by AdmiringWorm** [comment]
@supix sorry, had already merged the pull request before you commented that, mind opening a new pull request for those tuning changes?

**Pull request merged**